bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md

```diff
@@ -0,0 +1,2499 @@
+Fedipool, you could actually, but the big complaint that we see there is most miners are not gonna run their own nodes, they're not gonna be well-connected nodes, they're not gonna be constructing their own blocks, there's a lot of friction there.
+In a Fedipool, we could have a kind of hybrid model where the people that are running the federation servers, let's say it's seven individual miners and the quorum is a five of seven or something like that.
+Those seven individual miners that are running the Federation servers could also be doing the job negotiation and choosing which transactions are included in a block.
+So then the smaller miners that are part of that Fedipool, instead of imagining all the smaller miners all running their own nodes and doing this job negotiation and the latency that gets included with that whole process and whatnot, they could just kind of delegate who they want to do the transaction selection to those Federation members.
+So you could have like seven different members in a pool, basically all proposing block templates, proposing which transactions are included, and then a bunch of individual miners then choosing which of those seven they trust with the job construction.
+And so then all of a sudden, you have competition within a pool on who is constructing the transaction rather than having to completely leave the pool and go and switch to a different pool because you like how they choose the transaction.
+So you actually have this cool little competition, a game theory dynamic that happens internal within the pool.
+Does that make sense?
+
+NVK: 01:50:00
+
+That was great.
+
+Eric: 01:50:02
+
+Yeah.
+Yeah.
+And I just want to add, like, we shouldn't stop at Fedipools.
+Like there's so many other applications that we could federate.
+And as Matt mentioned, like everything essentially that takes custody of your Bitcoin and then allows you to do something smarter with it that can be implemented as the FediMint module.
+And if there are hackers out there that are listening, please come and build modules.
+It would be awesome.
+And we just finished like a big materialization effort.
+And now it's much, much easier to just implement your own applications on Fedimint.
+
+NVK: 01:50:42
+
+Very cool guys.
+
+Odell: 01:50:43
+
+Now I have to drop.
+This conversation was great.
+Appreciate you all.
+Cheers.
+
+Eric: 01:50:48
+
+Cheers.
+
+NVK: 01:50:49
+
+Any final thoughts, Matt?
+
+Odell: 01:50:51
+
+I did a full write up on Fedipool on my blog, DiscreetLog.com.
+Check it out.
+
+NVK: 01:50:57
+
+Awesome.
+That was great.
+
+Rijndael: 01:50:58
+
+Thank you.
+Thank you for joining us.
+
+Odell: 01:50:59
+
+Thanks.
+
+Rijndael: 01:51:01
+
+All right.
+Now that Matt's gone, let's get into the real talk about privacy. I'm just kidding.
+
+NVK: 01:51:06
+
+Well, you have nothing to hide, so.
+
+## Scriptable ecash01:59:36 Nostr + ecash
```

Comment:
The AI transcriptions and the final construction of this document are automated processed that we always try to refine and make them better. Here you can see an example of a mistake in the YouTube chapters of the original source (you can confirm it by looking at the description of the video) where a chapter title was merged with the previous one. You've got the timestamp (01:59:36), so it's easy to place it in the correct position.